qiskit/pulse/schedule/conditioned_schedule.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Configured Schedule.
+"""
+from .experiment_condition import UserLoDict
+from .pulse_schedule import Schedule
+
+
+class ConditionedSchedule:
```

Comment:
Is this really required or could we just supply a list of schedule configuration dictionaries?